src/Core/CommandBus/Parser/CommandDefinitionParser.php

```diff
@@ -32,7 +32,7 @@
 /**
  * Parses commands and queries definitions
  *
- * @deprecated
+ * @deprecated Since 1.7.8.0, use CommandHandlerDefinitionParser instead.
```

Comment:
Don't forget to trigger a deprecated: https://devdocs.prestashop.com/1.7/development/coding-standards/#deprecations